spec/interactors/create_password_reset_token_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe CreatePasswordResetToken do
+  describe ".call" do
+    let(:user) { create(:confirmed_user, reset_token: nil) }
+
+    context "when successful" do
+      subject do
+        described_class.call(user: user)
+      end
+
+      it "sets the user's reset token" do
+        expect(subject.user.reset_token).to eq(subject.raw)
+      end
+
+      it "sets the user's reset_digest" do
+        expect(subject.user.reset_digest).to eq(subject.enc)
+      end
+
+      it "sets the user's reset_sent_at" do
+        expect(subject.user.reset_sent_at).to be_an(ActiveSupport::TimeWithZone)
```

Comment:
Does the interactor fail if they are not nil? I think that a user who has requested a password reset before might not have nil values, right? I think it would be best to check that the time is set properly (i.e. 24 hours from now, or whatever we are expecting).